end_to_end/test_generate_param_only_checkpoint.sh

```diff
@@ -112,7 +112,6 @@ load_parameters_path=${base_output_directory}/${decode_ckpt_run_id}/checkpoints/
 attention=dot_product ici_tensor_parallelism=${ici_tensor_parallelism} steps=50 \
 metrics_file=/tmp/${run_id}_metrics.txt async_checkpointing=false max_target_length=128 per_device_batch_size=1 \
 quantization=${quantization} \
-${model_params} \
```

Comment:
The test fails with an error message: `jaxlib.xla_extension.XlaRuntimeError: INTERNAL: the requested functionality is not supported` on A3 nodes. 

Defaulting to base params somehow fixes the issue, and it doesn't harm the test.

Reply:
Did you cut a ticket, etc for this with HLO? The purpose of this exercise is to be squashing bugs...

Reply:
Yes, b/339473974